lib/pages/media_view.dart

```diff
@@ -14,10 +14,11 @@ import '../widgets/bottom_modal.dart';
 /// View to interact with a media object. Zoom in/out, download, share, etc.
 class MediaViewPage extends HookWidget {
   final String url;
+  final String heroTag;
```

Comment:
It looks like dart supports nullable types.
So it might make more sense to use a `String?` here and leave it uninitialized than to initialize to empty string and remember to check, since the type checker will force you to check for null when using it.
https://dart.dev/null-safety

Reply:
Thanks good idea!